zsh/PKGBUILD

```diff
@@ -1,39 +1,56 @@
-# Maintainer: Pierre Schmitz <pierre@archlinux.de>
-# PGO Version: Laio Seman <laio [at] ieee.org>
+# CachyOS Maintainer: Roice Young <insanelydefault@proton.me>
+# CachyOS Contributor: Laio Seman <laio [at] ieee.org>
+# Arch Linux Maintainer: Pierre Schmitz <pierre@archlinux.de>
 
 pkgbase=zsh
 pkgname=('zsh')
 pkgver=5.9
-pkgrel=4
+pkgrel=6
 arch=('x86_64')
 url='https://www.zsh.org/'
 license=('custom')
 makedepends=('pcre' 'libcap' 'gdbm' 'yodl' 'gcc')
-source=("https://www.zsh.org/pub/zsh-${pkgver}"{,-doc}".tar.xz"{,.asc}
```

Comment:
Please do not remove the .asc file. See [PKGBUILD/Integrity](https://wiki.archlinux.org/title/PKGBUILD#Integrity) for details but the takeaway is that it is good practice to provide the digital signatures that upstream provides when distributing PKGBUILDs

Reply:
it is now not removed, would you prefer that I restore the validpgpkeys value as well?

Reply:
Yes.

Reply:
I restored this